build.gradle

```diff
@@ -0,0 +1,111 @@
+import groovy.json.JsonException
+import groovy.json.JsonSlurper
+
+buildscript {
+    ext.kotlin_version = '1.2.0'
+
+    repositories {
+        mavenCentral()
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+        classpath 'com.moowork.gradle:gradle-node-plugin:1.2.0'
+    }
+}
+
+def dryRun = hasProperty("dryRun") ? property("dryRun") : "false"
+def authToken = hasProperty("authToken") ? property("authToken") : ""
+def deployTag = hasProperty("deployTag") ? property("deployTag") : "dev"
+
+subprojects.stream().filter { it.parent != rootProject }.each { p ->
+    configure(p) {
+        apply plugin: 'kotlin2js'
+        apply plugin: 'com.moowork.node'
+
+        node {
+            download = true
+            workDir = file("${rootProject.buildDir}/nodejs")
+        }
+
+        repositories {
+            mavenCentral()
+        }
+
+        dependencies {
+            compile "org.jetbrains.kotlin:kotlin-stdlib-js"
```

Comment:
Shouldn't we specify version explicitly?

Reply:
Doen